eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -12,10 +12,17 @@ module Test.Cardano.Ledger.Conway.CDDL (conway) where
 import Codec.CBOR.Cuddle.Huddle
 import Data.Function (($))
 import Data.Semigroup ((<>))
-import Data.Text qualified as T
 import Data.Word (Word64)
 import GHC.Num (Integer)
-import GHC.Show (Show (show))
+import Test.Cardano.Ledger.Core.Binary.CDDL
```

Comment:
We've just made a release, so addition of a new module will require a minor version bump for:
* `cardano-ledger-shelley-1.13.1.0`
* `cardano-ledger-conway-1.16.2.0`
* `cardano-ledger-core-1.14.1.0`

Also, the lower bounds will need to be adjusted accordingly